demo/predictions/css/demo.css

```diff
@@ -0,0 +1,45 @@
+:root {
+    --color-path-executed: #aea3a3;;
+}
+
+    /* parallel gateway icon */
+.bpmn-parallel-gateway.state-already-executed > path:nth-child(2),
+    /* message flow arrow */
+.bpmn-message-flow.state-already-executed > path:nth-child(4),
+    /* sequence flow arrow */
+.bpmn-sequence-flow.state-already-executed > path:nth-child(3) {
+    fill: var(--color-path-executed);
+}
+
+    /* message flow start marker */
+.bpmn-message-flow.state-already-executed > ellipse,
+    /* message flow line */
+.bpmn-message-flow.state-already-executed > path:nth-child(2),
+    /* message flow arrow */
+.bpmn-message-flow.state-already-executed > path:nth-child(4),
+    /* sequence flow line */
+.bpmn-sequence-flow.state-already-executed > path:nth-child(2),
+    /* sequence flow arrow */
+.bpmn-sequence-flow.state-already-executed > path:nth-child(3),
+    /* task */
+.bpmn-task.state-already-executed > rect,
+    /* parallel gateway stroke */
+.bpmn-parallel-gateway.state-already-executed > path:nth-child(1),
+    /* parallel gateway icon */
+.bpmn-parallel-gateway.state-already-executed > path:nth-child(2),
```

Comment:
❓ the selector could probably be simplify by not using 'nth-child' as we are targetting specific bpmn related class everywhere.

Reply:
done in d38be80